### PR TITLE
New home

### LINKED ISSUE #27: Update site design
Update site design to sync with the product launch.

### PR DESCRIPTION
**Pull request by jgwerner**
Deployed from local. Legacy site is now in legacy branch.

Closes #27

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "include-media": "^1.4.9",
     "is-directory": "^0.3.1",
     "lint-staged": "^3.4.1",
-    "next": "latest",
+    "mdi": "^1.9.33",
+    "next": "beta",
```
Comment (raygesualdo) [inline package.json]: Why did we switch from `latest` to `beta`?

File: pages/index.js [hunk 2]
```diff
@@ -1,365 +1,262 @@
...
+        icon: 'mdi-code-braces'
+    },
+    {
+        title: 'Business Analysts',
+        desc: 'Contribute, review and share data stories with all stakeholders.',
+        icon: 'mdi-chart-areaspline'
+    },
+    {
+        title: 'Software Engineers',
+        desc: 'Integrate assets with existing pipelines and applications using full featured API.',
```
Comment (raygesualdo) [inline pages/index.js]: sp: "using a full-featured API."

File: pages/index.js [hunk 3]
```diff
@@ -1,365 +1,262 @@
...
+        icon: 'mdi-server'
+    },
+    {
+        title: 'Executives',
+        desc: 'Accelerate data driven organizations with best of breed tools and services.',
```
Comment (raygesualdo) [inline pages/index.js]: sp: "data-driven"

File: pages/index.js [hunk 4]
```diff
@@ -1,365 +1,262 @@
...
+                <ContentSection graphite className="special-mock-content-section">
+                    <ContentPane half center-vertically>
+                        <Title>
+                            What is 3Blades?
+                        </Title>
+                        <Body>
+                        <h2 className="special-subtitle"> 3Blades offers a cloud based development environment to build and share data science
+                            projects.</h2>
+                        <p>
+                            Guided data science are services used by business analysts and data scientists to
```
Comment (raygesualdo) [inline pages/index.js]: Subject/verb agreement feels wonky. Suggestion: "Guided data science is a collection of services used" OR "Guided data science is a suite of services used".

File: pages/index.js [hunk 5]
```diff
@@ -1,365 +1,262 @@
...
+                            enhance their productivity while exploring data, building machine and deep learning models,
+                            and creating rich visualizations to tell their data stories.
+                        </p>
+                        <p>
+                            3Blades data scienctists have created a set of interactive helper services to guide the user with these tasks with the goal of enhancing
```
Comment (raygesualdo) [inline pages/index.js]: Suggestion: "interactive helper services that increase business value by guiding the user in these tasks."

File: pages/index.js [hunk 6]
```diff
@@ -1,365 +1,262 @@
...
-          <ContentPane half center-vertically>
-            <Body>
-              <Graphic image="/static/images/hero-graphic.png" />
-            </Body>
-          </ContentPane>
-        </ContentSection>
+                <ContentSection gold-gradient>
+                    <ContentPane full center center-vertically>
+                        <Title>
+                            Who Uses 3Blades?
```
Comment (raygesualdo) [inline pages/index.js]: Inconsistent capitalization. Suggestion: "Who uses 3Blades?"

File: pages/index.js [hunk 7]
```diff
@@ -1,365 +1,262 @@
...
-      </Layout>
-    )
-  }
+                <ContentSection className="features-section">
+                    <ContentPane half-featured>
+                        <Title>
+                            Platform Resources
+                        </Title>
+                        <Body>
+                        <p>Obtain an introduction on how to use the 3Blades platform or dive deeper with advanced topics or interactive API docs.</p>
```
Comment (raygesualdo) [inline pages/index.js]: Suggestion: "View the 3Blades platform introduction, dive deeper into advanced topics, or experience our interactive API docs."

File: package.json (hunk 1 above)
Comment (jgwerner) [inline package.json]: Changed to latest, not sure why that was changed.

**Comment by jgwerner** [comment]
Change requests pushed.

File: pages/index.js [hunk 8]
```diff
@@ -74,12 +74,12 @@ const whoList = [
     },
     {
         title: 'Software Engineers',
-        desc: 'Integrate assets with existing pipelines and applications using full featured API.',
+        desc: 'Integrate assets with existing pipelines and applications using full-featured API.',
```
Comment (raygesualdo) [inline pages/index.js]: sp: "using *a* full-featured API."

File: pages/index.js [hunk 9]
```diff
@@ -205,13 +205,12 @@ export default class Index extends React.Component {
...
                         <p>
-                            Guided data science are services used by business analysts and data scientists to
+                            Guided data science consists of services used by business analysts and data scientists to
                             enhance their productivity while exploring data, building machine and deep learning models,
                             and creating rich visualizations to tell their data stories.
                         </p>
                         <p>
-                            3Blades data scienctists have created a set of interactive helper services to guide the user with these tasks with the goal of enhancing
-                            business value.
+                            3Blades data scientists have created a set of interactive helper services that increase business value by guiding the user in these tasks with the goal of enhancing business value.
```
Comment (raygesualdo) [inline pages/index.js]: sp: "3Blades data scientists have created a set of interactive helper services that increases business value by guiding the user in these tasks."

**Pull request merged**